pallets/messages/src/types.rs

```diff
@@ -8,6 +11,12 @@ use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
 use scale_info::TypeInfo;
 extern crate alloc;
 use alloc::vec::Vec;
+use common_primitives::messages::MessageResponseV2;
+
+/// Multihash hash function code for SHA2-256
+pub const SHA2_256: u64 = 0x12;
+/// Multihash hash function code for Blake3
+pub const BLAKE3: u64 = 0x1e;
```

Comment:
Move this to common? so msa pallet can use it too (i might add it already in my PR)